packages/shared/src/modules/user-follows/libs/types/types.ts

```diff
@@ -0,0 +1,3 @@
+export { type UserFollowsFollowParametersDto } from "./user-follows-follow-parameters-dto.types.js";
+export { type UserFollowsFollowRequestDto } from "./user-follows-follow-request-dto.types.js";
+export { type UserFollowsUnfollowParametersDto } from "./user-follows-unfollow-parameters-dto.types.js";
```

Comment:
Other than this, LGTM
```suggestion
export { type UserFollowsFollowParametersDto } from "./user-follows-follow-parameters-dto.type.js";
export { type UserFollowsFollowRequestDto } from "./user-follows-follow-request-dto.type.js";
export { type UserFollowsUnfollowParametersDto } from "./user-follows-unfollow-parameters-dto.type.js";
```

Reply:
Refactored